waiter/src/waiter/scheduler.clj

```diff
@@ -86,12 +85,6 @@
 
 (defprotocol ServiceScheduler
 
-  (get-service->instances [this]
```

Comment:
This method no longer needs to be part of the scheduler interface. Instead, it moves to a separate interface which allows the scheduler to express interest in being used as a 'pollable' scheduler.